source/application/SGM/SGMAppController.cpp

```diff
@@ -1489,10 +1489,18 @@ bool SGMAppController::setupSGMPeriodicTable(){
 	return success;
 }
 
+#include "ui/AMMotorGroupView.h"
```

Comment:
I know this was just for testing, but could you move this to the top of the file with the rest of the includes?